lib/HLSL/DxilOperations.cpp

```diff
@@ -718,6 +727,15 @@ void OP::RemoveFunction(Function *F) {
   }
 }
 
+OP::OpCodeClass OP::GetOpClassForDxilFunction(const Function *F) {
+  auto iter = m_FunctionToOpClass.find(F);
+  if (iter == m_FunctionToOpClass.end()) {
```

Comment:
Why not just DXASSERT(iter != m_FunctionToOpClass.end(), "should..."); and then keep running.